2-mandatory/4-stocks.js

```diff
@@ -47,10 +50,19 @@ function getAveragePrices(closingPricesForAllStocks) {
                 (Apple's price on the 5th day) - (Apple's price on the 1st day) = 172.99 - 179.19 = -6.2
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
-function getPriceChanges(closingPricesForAllStocks) {
-    // TODO
-}
-
+function calculateChange (prices){
+    return (prices[prices.length-1]-prices[0]).toFixed(2)
+ }
+ 
+ function getPriceChanges(closingPricesForAllStocks) {
+     let changArr = []
+     for ( let i of closingPricesForAllStocks){
+         let change = calculateChange(i)
+         changArr.push(Number(change))
+     }
+     return changArr 
```

Comment:
When an array has an input with same size as the output you can also use `map` but also like this works perfectly

Reply:
thank you, switched to map